pipeline_dp/dp_engine.py

```diff
@@ -106,6 +106,7 @@ def _drop_not_public_partitions(self, col, public_partitions,
             "Extract partition id")
         col = self._ops.filter_by_key(col, public_partitions,
                                       "Filtering out non-public partitions")
+        self._add_report_stage(f"Dropped non public partitions")
```

Comment:
Maybe to have format for all messages like that:
: (for contribution bounding it's the case).

Here it might be like
**Public partition selection: dropped non public partitions**

WDYT?

Reply:
Yes, this looks great. Completed!